### ISSUE TITLE
Python buildpack fails with error: '/app/.heroku/python/bin/pip: No such file or directory'

### ISSUE THREAD

**Issue by itsMichaelBates** [flynn/flynn#4143]
I just started using Flynn and deployed three nodes in a cluster running `v20170514.0-a013e9a` on AWS using Ubuntu 16.04. The setup process worked fine and I was able to create an app.

I'm using two buildpacks - the contents of my `.buildpacks` file is:

```
https://github.com/heroku/heroku-buildpack-nodejs.git
https://github.com/heroku/heroku-buildpack-python.git
```

When I try to deploy my app, I get the following error:

```
-----> Building inl...
-----> Multipack app detected
       =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-nodejs.git
       =====> Detected Framework: Node.js
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NPM_CONFIG_PRODUCTION=true
       NODE_VERBOSE=false
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 6.x via semver.io...
       Downloading and installing node 6.11.0...
       Using default npm version: 3.10.10
       
-----> Restoring cache
       Skipping cache restore (new-signature)
       
-----> Building dependencies
       Installing node modules (package.json)
       
       > node-sass@4.5.3 install /tmp/build/app/node_modules/node-sass
       > node scripts/install.js
       
       Downloading binary from https://github.com/sass/node-sass/releases/download/v4.5.3/linux-x64-48_binding.node
       Download complete
       Binary saved to /tmp/build/app/node_modules/node-sass/vendor/linux-x64-48/binding.node
       Caching binary to /tmp/app/.npm/node-sass/4.5.3/linux-x64-48_binding.node
       
       > node-sass@4.5.3 postinstall /tmp/build/app/node_modules/node-sass
       > node scripts/build.js
       
       Binary found at /tmp/build/app/node_modules/node-sass/vendor/linux-x64-48/binding.node
       Testing binary
       Binary is fine
       inl@2.0.0 /tmp/build/app
       +-- angular@1.6.4
       ... <lots of other npm modules>
       
       
-----> Caching build
       Clearing previous node cache
       Saving 2 cacheDirectories (default):
       - node_modules
       - bower_components (nothing to cache)
       
-----> Build succeeded!
       =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-python.git
       =====> Detected Framework: Python
-----> Installing python-3.4
-----> Installing pip
-----> Installing requirements with pip
       /tmp/buildpackOPjlB/bin/steps/pip-install: line 7: /app/.heroku/python/bin/pip: No such file or directory
remote: ERROR: Build failed: exec: job exited with status 1
```

It looks like the pip executable cannot be found. Am I doing something wrong?

Thanks

**Comment by itsMichaelBates** [comment]
I was able to fix this by changing my `runtime.txt` config. I was using `python-3.4` but I changed to `python-3.6.1` and that fixed the problem.

Not sure what's going on there, but I actually prefer Python 3.6.1. I was just following the docs at https://flynn.io/docs/languages/python which say:

> The latest python-2.7 and python-3.4 are officially supported, but any runtime between 2.4.4 3.4.1 can be used, including PyPy runtimes.

**Comment by titanous** [comment]
Looks like this was a docs issue fixed in #4141 but didn't make it into the website. I've deployed an update.
